### PR TITLE
Rebind dropdowns when switching to uk address

### PR DESCRIPTION
**Pull request by dougajmcdonald**
This resolves an issue when navigating back to the address page from the 'change' link during the signup process

### CODE REVIEW

File: GLAA.Web/wwwroot/sass/glaa.scss [hunk 1]
```diff
@@ -307,7 +307,7 @@ table.publicrreigster tbody td {
 }
 
 .public-register-selected-country {
-    background-image: url("/images/icon-button-remove.png");
+    background-image: url("/public/images/icon-button-remove.png");
```
Comment (SeiyaYamamotoStudios) [inline GLAA.Web/wwwroot/sass/glaa.scss]: Ah! Good to see this fix 👍

**Review by SeiyaYamamotoStudios — APPROVED** [review]
I was wondering if some of the country population in the controller could be in the ViewModelBuilder - but maybe it isn't worth doing at this stage.

Nice to see the "remove" icon going back in!

File: GLAA.Web/wwwroot/sass/glaa.scss (hunk 1 above)
Comment (dougajmcdonald) [reply GLAA.Web/wwwroot/sass/glaa.scss]: What I don't understand is why the build process in webpack didn't seem to pickup your image file properly

**Comment by dougajmcdonald** [comment]
Well, in the event of failed validation the viewmodelbuilder isn't called. That's the problem. 
The builder does create the lookups but the data isn't posted back so isn't bound the model again on post, longer term we may want to change this approach and have that data posted.

**Pull request merged**